### PR TITLE
[15.0][MIG] sale_missing_cart_tracking_tier_validation > sale_missing_tracking_tier_validation: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT40150

Pending from:
- [x] #2339 

@sergio-teruel @chienandalu please review :)

### CODE REVIEW

File: sale_missing_tracking_tier_validation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Tier Validation",
+    "summary": "Extends the functionality of Sale missing tracking exceptions to "
+    "support a tier validation process.",
+    "version": "15.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```
Comment (sergio-teruel) [inline sale_missing_tracking_tier_validation/__manifest__.py]: Change author to Tecnativa

File: sale_missing_tracking_tier_validation/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from . import sale_missing_cart_tracking_exception
```
Comment (sergio-teruel) [inline sale_missing_tracking_tier_validation/models/__init__.py]: remove cart

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by pilarvargas-tecnativa** [comment]
@sergio-teruel changes made, thanks for suggestions :)

**Comment by pilarvargas-tecnativa** [comment]
@chienandalu can you check this please?

**Comment by pilarvargas-tecnativa** [comment]
@carlosdauden can you take a look?

File: sale_missing_tracking_tier_validation/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Tecnativa - Carlos Dauden
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Tier Validation",
```
Comment (victoralmau) [inline sale_missing_tracking_tier_validation/__manifest__.py]: Perhaps the name could be more explanatory (although the description already is).

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by chienandalu — COMMENTED** [review]
Would you push again to have a runboat?

File: sale_missing_tracking_tier_validation/__manifest__.py (hunk 3 above)
Comment (pilarvargas-tecnativa) [reply sale_missing_tracking_tier_validation/__manifest__.py]: thanks :)

**Comment by pilarvargas-tecnativa** [comment]
> Would you push again to have a runboat?

There it is! :smiley_cat:

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Pull request merged**